SMSEagleIOTBridge/Readme_Prepare_Eagle.md

```diff
@@ -149,11 +149,44 @@ chmod 644 /etc/systemd/system/nyss-iot-bridge.service
 chmod +x /home/pi/smsEagle-iot-hub-handler.py
 ```
 
+### 2.4. Download and run setup script
+
+This repository also contains a setup shell script that can be downloaded to skip steps 2.3-2.4
```

Comment:
The script replaces the steps 2.1-2.4. I would put this at the top of section 2 and say that if the script doesn't work, then you need to follow the rest of the steps in section 2.